pages/pricing.module.scss

```diff
@@ -0,0 +1,4 @@
+.pricingList {
```

Comment:
@krishnaglick recommended we move towards using modular SCSS for items that are only used on a single page, so I've started us out.